codestodians/food-basket-front-end
# [Chore: #154141205] Rename non-component files

#### What does this PR do?
- Renames non-component files to use lowercase
- Refactors import statments due to directory and file renaming

#### Description of Task to be completed?
- Rename non-component files to use the lowercase naming convention.
- Rename `CustomComponents` directory.
- Refactor error message in authentication based on change on the backend

#### How should this be manually tested?
- Check out into the `ch-rename-non-component-files-154141205` branch.
- Run the app as well as tests to ensure everything is still in order.

#### Any background context you want to provide?
- As regards the renaming of the `CustomComponents` directory, I realized that all components on the App are actually 'custom' components, since they are customized by us. Also, I think that renaming to say `base` provides shorter import statements for our "base components" when importing them for use anywhere.
- I also find it necessary that only Components are named with uppercase. This is a convention that helps with easy identification 

#### What are the relevant pivotal tracker stories?
- [#154141205](https://www.pivotaltracker.com/story/show/154141205)

#### EXTRA DETAILS
- The sheer volume in the number of files changed is as a result of the renaming involved. Git sees filed renamed as deleted and then creates new files, thus leading to the grand illusion.
